src/routes/api/users.js

```diff
@@ -114,8 +116,83 @@ const router = express.Router();
  *       '404':
  *         description: User no registered!
  */
+/**
+ * @swagger
+ * definitions:
+ *   forgotpassword:
+ *     type: object
+ *     properties:
+ *       email:
+ *         type: string
+ *         example: krinkun09@gmail.com
+ */
+/**
+ * @swagger
+ * /forgotpassword:
+ *   post:
+ *     tags:
+ *       - Authentication
+ *     name: Forgotpassword
+ *     summary: Send link on Email
+ *     consumes:
+ *       - application/json
+ *     produces:
+ *       - application/json
+ *     parameters:
+ *       - name: body
+ *         in: body
+ *         schema:
+ *          $ref: '#/definitions/forgotpassword'
+ *     responses:
+ *       '200':
+ *         description: please check your email to see the link for reseting password
+ *       '400':
+ *         description: can not find that user
+ */
+/**
+ * @swagger
+ * definitions:
+ *   resetpassword:
+ *     type: object
+ *     properties:
+ *       email:
+ *         type: string
+ *         example: krinkun09@gmail.com
+ *       password:
+ *         type: string
+ *         format: string
+ *         example: Pa55W0rd
+ */
+/**
+ * @swagger
+ * /resetpassword:
```

Comment:
swagger route should be **users/resetpassword/{token}** and have a parameter **"in:path"** for users to be able to pass in the **token**
Look at Eric's swagger doc for the verify token